src/Core/AdviceMatcher.php

```diff
@@ -166,6 +166,11 @@ private function getAdvicesFromAdvisor(
 
             $mask = ReflectionMethod::IS_PUBLIC | ReflectionMethod::IS_PROTECTED;
             foreach ($class->getMethods($mask) as $method) {
+                // abstract methods can not be weaved
```

Comment:
> One more caution: modifier filter is also used for property matching. Of course, "not abstract" for properties will be always true. But if we will use ModiferFilter for properties with negation, then it can be very unpredictable :)

You are soooo right! Ok, how about this? Simply - advice matcher will skip abstract methods for weaving, regardless of pointcut expression used?